### PR TITLE
feat(avatar-component): Avatar component CSS

### PR DESCRIPTION
**Pull request by anupkrbid**
This is just the CSS for the avatar component.
Proper doc is not added will be added once the P0 components are ready. There is an index .html present to temporarily test the implementation

Deploy URL:  https://deploy-preview-2--totality-ui.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    <small>P 16px / Line Height 16px</small>
+
+    <p class="has-fs-xsm has-lh-xsm">I am a (XSM) Paragraph</p>
+    <small>P 8px / Line Height 8px</small>
+
+    <hr />
+    <h1>Avatar</h1>
+    <hr />
+    <img
+      class="avatar is-sm"
```
Comment (viru2001) [inline index.html:65]: you can give classname avatar-sm instead of is-sm

File: src/css/styles.css [hunk 2]
```diff
@@ -57,12 +57,28 @@ p {
...
+ * 1. Avatar
+ */
+
+/* Avatar */
+.avatar {
+  display: inline-block;
+  vertical-align: middle;
+  font-size: var(--t-font-size-md);
+  block-size: 2.5em;
+  inline-size: 2.5em;
```
Comment (viru2001) [inline src/css/styles.css:72]: you can give this size in avatar-lg , avaatr-md classes ... your avatar class should be generic

File: src/css/styles.css [hunk 3]
```diff
@@ -124,3 +140,33 @@ p {
 .has-fw-black {
```
Comment (viru2001) [inline src/css/styles.css:169]: you can give classname here as `font-wt-black` or `fw-black`

File: src/css/styles.css [hunk 4]
```diff
@@ -124,3 +140,33 @@ p {
...
   font-weight: var(--font-weight-black) !important;
 }
+
+/* General Sizing */
+
+.is-xsm {
+  font-size: var(--t-font-size-xsm) !important;
```
Comment (viru2001) [inline src/css/styles.css:176]: you can remove `!important` ... there is no need of that ... are you facing any problem without using it ?

File: index.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      src="https://robohash.org/127539fe860f082d7fb9e34dbf58d4ed?set=set4&bgset=bg1&size=400x400"
+      alt="avatar"
+    />
+    <img
+      class="avatar is-lg"
+      src="https://robohash.org/127539fe860f082d7fb9e34dbf58d4ed?set=set4&bgset=bg1&size=400x400"
+      alt="avatar"
+    />
+    <img
+      class="avatar is-sm is-rounded"
```
Comment (viru2001) [inline index.html:80]: you can give classname as `rounded`

**Review by viru2001 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (anupkrbid) [reply index.html:65]: I liked the naming convention for [Bulma](https://bulma.io/) they use this convention so I'm following that. so `.is-sm` is a utility class that can be added to anything whose size needs to be increased/decreased. so if I give Avalar specific classes then the same CSS properties will be there in different classes.

File: index.html (hunk 5 above)
Comment (anupkrbid) [reply index.html:80]: here also I'm using the utility class naming pattern in https://bulma.io/

File: src/css/styles.css (hunk 2 above)
Comment (anupkrbid) [reply src/css/styles.css:72]: Here I have a generic class for avatar and I'm using utility class to modify it..size and border radius

File: src/css/styles.css (hunk 4 above)
Comment (anupkrbid) [reply src/css/styles.css:176]: No, I'm not facing any issues. I am following the utility pattern in Bootstrap. Which uses utility class to overwrite any existing single style.

File: index.html (hunk 5 above)
Comment (viru2001) [reply index.html:80]: ok ... got it

File: src/css/styles.css (hunk 2 above)
Comment (viru2001) [reply src/css/styles.css:72]: ok

File: index.html [hunk 6]
```diff
@@ -10,6 +10,10 @@
       name="description"
       content="Totality is a simple CSS frameowrk to kick start your frontend project"
     />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"
+      rel="stylesheet"
+    />
```
Comment (viru2001) [inline index.html]: you can import this in your main CSS file. so that you don't need to import it in each page in html

**Review by viru2001 — COMMENTED** [review]
suggested one minor change . rest everything looks good to me

File: index.html (hunk 6 above)
Comment (anupkrbid) [reply index.html]: yes okay

**Pull request merged**